capi-matrix/README.md

```diff
@@ -0,0 +1,52 @@
+# RFC 0015 - CAPI matrix per provider
+
+## Introduction
+
+We have a cluster-<provider> chart for every provider that we support (it is our UI to the customer). It has a set of templates that allow our customers to create a cluster app with all necessary resources required by a Cluster API implementation. 
```

Comment:
"it is our UI to the customer" sounds like this is THE user interface, as if there are no others.

I guess you'd like to express something like: The cluster app is our technical interface to configure clusters.

In terms of terminology, I would replace "chart" with "app".

Reply:
yeah good point. Adding it